tabpfn_client/estimator.py

```diff
@@ -212,8 +212,8 @@ def fit(self, X, y):
         if config.g_tabpfn_config.use_server:
             try:
                 assert (
-                    self.model == "latest_tabpfn_hosted"
```

Comment:
Could you please also remove this assert and add the other models, like you did for regression?

Reply:
Okay done :)